WcaOnRails/app/webpacker/lib/requests/routes.js.erb

```diff
@@ -142,6 +141,11 @@ export const competitionsForSeniorUrl = (userId) => `<%= CGI.unescape(Rails.appl
 export const adminCheckRecordsUrl = `<%= CGI.unescape(Rails.application.routes.url_helpers.admin_check_regional_records_path)%>`;
 
 export const apiV0Urls = {
+  users: {
+    me: {
+      permissions: () => `<%= CGI.unescape(Rails.application.routes.url_helpers.api_v0_users_me_permissions_path) %>`,
```

Comment:
why is this a method and not just a constant?

Reply:
Actually wanted to maintain everything as method. But had a similar discussion in another PR for a similar thing, so changed it to constant.